src/krux/pages/qr_view.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, ctx, binary=False, data=None, title=None):
                 self.title = t("Compact SeedQR")
                 self.code = self._binary_seed_qr()
             else:
-                self.title = t("SeedQR")
+                self.title = "SeedQR"
```

Comment:
no translations for "SeedQR", but translations for "Compact SeedQR"?

Reply:
Good point! I didn't changed because a lot of languages translated this In portuguese "compact" means "compacto", and it makes sense to me. What do you think about @odudex ?